archive/app/controllers/ArchiveController.scala

```diff
@@ -64,7 +64,7 @@ class ArchiveController(dynamoDB: DynamoDB) extends Controller with Logging with
         Some(path)
       case _ => None
     }
-    s"http://${normalised.getOrElse(path)}"
+    s"https://${normalised.getOrElse(path)}"
```

Comment:
I 'think' the above code is just to make it work on different environments. LinkTo will achieve the same thing, but be a bit cleaner maybe